### ISSUE TITLE
WARNING  Automatic initialisation based on OS detection

### ISSUE THREAD

**Issue by ocroquette** [MatthieuDartiailh/pyclibrary#48]
I get the following warning logged when I used pyclibrary:

```
WARNING  pyclibrary.c_parser:c_parser.py:418 Automatic initialisation based on OS detection
````

However, the automatic initialization seems to work fine since I never has any issue.

Does the warning still make sense ? If yes, how to prevent it ?

**Comment by MatthieuDartiailh** [comment]
This warning is indeed a bit excessive, logging at the info would be sufficient. I will push the change.

**Comment by MatthieuDartiailh** [comment]
Addressed in 074cbcd5

**Comment by ocroquette** [comment]
Thank you !
